x-pack/plugins/maps/public/content_management/maps_client.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { SearchQuery } from '@kbn/content-management-plugin/common';
+
+import type {
+  MapGetIn,
+  MapGetOut,
+  MapCreateIn,
+  MapCreateOut,
+  MapUpdateIn,
+  MapUpdateOut,
+  MapDeleteIn,
+  MapDeleteOut,
+  MapSearchIn,
+  MapSearchOut,
+  MapSearchOptions,
+} from '../../common/content_management';
+import { getContentManagement } from '../kibana_services';
+
+const get = async (id: string) => {
+  return getContentManagement().client.get<MapGetIn, MapGetOut>({
+    contentTypeId: 'map',
```

Comment:
any way we can make this a constant?

e.g.:
https://github.com/elastic/kibana/blob/35b5dd38f3e3dea74d28391385944239006b3610/x-pack/plugins/maps/common/constants.ts#L11

```export const CONTENT_MANAGEMENT_TYPE = 'map';```

Reply:
There is no need to create a constant, it is already a constant by using a type, with the benefit that it does not add anything to the js bundle. Also it is abstracted inside this single `maps_client.ts` file so it isn't used anywhere else.

```ts
export type MapContentType = 'map';
```

And no other value can be passed
<img width="792" alt="Screenshot 2023-04-13 at 10 25 45" src="https://user-images.githubusercontent.com/2854616/231716697-f7ee917f-cf2a-4431-9eb1-a7fc4f931039.png">